app/components/NavButtons/index.js

```diff
@@ -5,13 +5,25 @@ import FontAwesome from 'react-fontawesome';
 import styles from './styles.scss';
 
 const NavButtons = props => {
+
+  let enableBackButton = true;
+  let enableForwardButton = true;
+
+  if (props.historyCurrentIndex == 1) {
+    enableBackButton = false;
+  }
+
+  if (props.historyCurrentIndex == (props.historyLength - 1)) {
+    enableForwardButton = false;
+  }
+
   return (
     <div className={styles.nav_buttons}>
-      <a href='#' onClick={props.back} >
-        <FontAwesome name='chevron-left'/>
+      <a href='#' onClick={(enableBackButton ? props.back : null)} className={(enableBackButton ? null : 'disable' )}>
```

Comment:
instead of using trinaries, try using && and classnames module, e.g.:

```javascript
//at the top
import cx from 'classnames';
// ...
<a href='#' onClick={enableBackButton && props.back} className={cx({'disable': !enableBackButton})}>
```